src/test/java/com/inqoo/trainingservice/app/service/CustomerServiceIT.java

```diff
@@ -0,0 +1,65 @@
+package com.inqoo.trainingservice.app.service;
+
+
+import com.inqoo.trainingservice.app.customer.*;
+import com.inqoo.trainingservice.app.exception.NameAlreadyTakenException;
+import com.inqoo.trainingservice.app.trainer.Trainer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@Transactional
+public class CustomerServiceIT {
```

Comment:
Please add test cases for happy paths also

Reply:
done